### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by Juanse7793**
## Here I:

- Make it look great :smiley: 

## Requirements

- I use the information detailed in Zeplin to style the website. :nerd_face: 
- My final design matches Zeplin's design. :heavy_check_mark:

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -0,0 +1,312 @@
+.add-title {
+  color: #888;
+  width: 10.875rem;
+  height: 1.5rem;
+  margin: 1.813rem 32.75rem 1.188rem 0;
+  font-family: Montserrat, sans-serif;
```
Comment (Gambit142) [inline src/App.css]: - [x] I can see you repeated this CSS declaration many times. I would like to advise you to put it in a **_class_** and use the **_className_** on elements. For example:
```
.montserrat {
  font-family: Montserrat, sans-serif;
}
```
Then use it on elements `<div className='montserrat'>Hello World</div>`.

Alternatively, you can use CSS variables. You can read this [article](https://www.w3schools.com/css/css3_variables.asp) to learn more about CSS variables. 👍

File: src/App.css [hunk 2]
```diff
@@ -0,0 +1,312 @@
...
+  font-size: 1.25rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
```
Comment (Gambit142) [inline src/App.css]: - [x] Kindly refer to the first comment https://github.com/Juanse7793/Bookstore/pull/5#discussion_r866353581 for these three CSS declarations repeated many times. 👍

File: src/App.css [hunk 3]
```diff
@@ -0,0 +1,312 @@
...
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.title-change {
+  font-family: 'Roboto Slab', sans-serif;
```
Comment (Gambit142) [inline src/App.css]: - [x] Kindly refer to the first comment https://github.com/Juanse7793/Bookstore/pull/5#discussion_r866353581 for this CSS declaration for the `Roboto` font 👍

File: src/App.css [hunk 4]
```diff
@@ -0,0 +1,312 @@
...
+  margin: 5px 0 25px 0;
+  font-family: 'Roboto Slab', sans-serif;
+  font-size: 1rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.4px;
+  color: #121212;
+}
```
Comment (Gambit142) [inline src/App.css]: - [x] Kindly refer to the first comment https://github.com/Juanse7793/Bookstore/pull/5#discussion_r866353581 for the color `#121212` used many times. 👍

File: src/App.css [hunk 5]
```diff
@@ -0,0 +1,312 @@
...
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.15px;
+  color: #121212;
+  padding: 0 0.8rem;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+}
```
Comment (Gambit142) [inline src/App.css:253]: - [x] Kindly refer to the first comment https://github.com/Juanse7793/Bookstore/pull/5#discussion_r866353581 for the color `#e8e8e8` used many times. 👍

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @Juanse7793 👋, 

Good job so far!

## Highlights

- Your Design is very identical to that of Zeplin's template provided. 💯 

However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
#### _Kindly check the comments under the review and apply the changes to all stylesheets._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/pages/navbar.js [hunk 6]
```diff
@@ -1,14 +1,18 @@
...
     </div>
     <div className="navbar-links">
-      <a className="links" href="/">Books</a>
-      <a className="links" href="/categories">Categories</a>
+      <a className="links-books" href="/">BOOKS</a>
+      <a className="links-categories" href="/categories">CATEGORIES</a>
+    </div>
+    <div className="circle-person">
+      <Icon className="icon" icon="bi:person-fill" />
     </div>
```
Comment (FrederickMih) [inline src/pages/navbar.js:16]: - [Optional] Please align this icon with the main container as seen in the screenshots below:

Your Implementation | Zeplin Design
:---:|:---:
|

**Review by FrederickMih — APPROVED** [review]
# Status: Approved :tada: :tada: 

Hi @Juanse7793 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/pages/navbar.js (hunk 6 above)
Comment (Juanse7793) [reply src/pages/navbar.js:16]: Done :100:

**Pull request merged**